rtlib/generic_opts.hh

```diff
@@ -99,23 +151,55 @@ class Opts {
 #ifdef LIBRNA_RNALIB_H_
         << " (-[tT] [0-9]+)? (-P PARAM-file)?"
 #endif
-        << " (-[drk] [0-9]+)* (-h)? (INPUT|-f INPUT-file)\n";
-    }
+        << " (-[drk] [0-9]+)* (-h)? (INPUT|-f INPUT-file)\n"
+        << "--help,-H,-h                          print this help message\n"
+#ifdef CHECKPOINTING_INTEGRATED
+        << "--checkpointInterval,-c    d:h:m:s    specify the checkpointing "
+        << "interval, default: 0:0:1:0 (1h)\n"
+        << "--checkpointOutput,-O      PATH       set the path where to store "
+        << "the checkpoints, default: current working directory\n"
+        << "                                      The program will also attempt"
+        << " to read existing checkpoints from a Logfile from this path.\n"
+        << "                                      Use the --checkpointInput "
+        << "option to explictly set the input path for all checkpoints.\n"
```

Comment:
should we reformat such that text does not exceed 80 chars when msg is displayed on the cmd?

Reply:
Done.